packages/prisma/src/prisma/migrations/20230726182929_rename_session/migration.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE "Session" RENAME TO "LiveSession";
```

Comment:
we need to rename the Session table to LiveSession as next-auth uses a Session table of its own -> first migration